capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -0,0 +1,112 @@
+document.body.onload = function() {
+  autocomplete(['Hello', 'Hi', 'Howdy', 'Yello']);
+};
+
+/**
+ * @param {string[]} arr
+ * @return {void}
+ * Attributing this code to: w3 schools
+ * Link: https://www.w3schools.com/howto/howto_js_autocomplete.asp
+*/
+function autocomplete(arr) {
+  const inp = document.getElementById('search');
+  let currentFocus = -1;
+  /* Event occurrance when input is provided to autocomplete field */
+  inp.addEventListener('input', function(e) {
+    closeAllLists();
+    const val = this.value;
+    if (!val) {
+      return false;
+    }
+    currentFocus = -1;
+    const autocompleteList = document.createElement('DIV');
```

Comment:
Make `'DIV'` lowercase for consistency with how you would actually write HTML.

Reply:
done!